src/test/java/org/primeframework/mvc/JWTRefreshTokenLoginTest.java

```diff
@@ -256,6 +258,29 @@ public void refreshTokenEndpointUp_no_auth() {
              .assertBodyContains("Logged in");
   }
 
+  @Test
+  public void refreshTokenEndpoint_additionalParameters() {
+    // The token action should get called with the additional parameters we configure on the security context.
+    // This will be validated in the TokenAction.
+    MockOAuthUserLoginSecurityContext.additionalParameters.put("tenantId", List.of(new UUID(5, 0).toString()));
+    MockOAuthUserLoginSecurityContext.TokenEndpoint = "http://localhost:" + simulator.getPort() + "/oauth/token";
+    MockOAuthUserLoginSecurityContext.ValidateJWTOnLogin = false;
+
+    // Setting 'expired: true' on the request just tells the Login action to create an expired JWT and store it in the LoginContext.
+    // - So we expect this to succeed, but the login context wil now contain an expired JWT. This means it will be refreshed on first use.
+    simulator.test("/oauth/login")
+             .withParameter("expired", "true")
```

Comment:
The `expired` stuff doesn't really have any thing to do with this test does it? Can we remove it so this test only asserts that if you pass in the additional parameters to the `*LoginSecurityContext` they are sent in the POST body request?

Reply:
We need that in order to trigger the token refresh on the next call. Added a comment to make that more obvious.